pkg/protocol/handshake/message_certificate_request_test.go

```diff
@@ -28,8 +29,8 @@ func TestHandshakeMessageCertificateRequest(t *testing.T) {
 			{Hash: hash.SHA512, Signature: signature.RSA},
 			{Hash: hash.SHA1, Signature: signature.RSA},
 		},
+		CertificateAuthoritiesNames: [][]byte{[]byte("test")},
```

Comment:
Please separate test case to check behavior for empty CA names as well.

Reply:
fixed